docs/docs/features-limitations.md

```diff
@@ -5,7 +5,7 @@ This section lists planned features and current limitations of Contrast.
 ## Availability
 
 - **Platform support**: At present, Contrast is exclusively available on Azure AKS, supported by the [Confidential Container preview for AKS](https://learn.microsoft.com/en-us/azure/confidential-computing/confidential-containers-on-aks-preview). Expansion to other cloud platforms is planned, pending the availability of necessary infrastructure enhancements.
-- **Bare metal support**: Support for running Contrast on bare metal Kubernetes will be available soon for AMD SEV and Intel TDX.
+- **Bare-metal support**: Support for running [Contrast on bare-metal Kubernetes](getting-started/bare-metal.md) is available for AMD SEV and Intel TDX.
```

Comment:
We use "bare metal" (with a space) in most (all?) other places. We should be consistent about this.

Reply:
In this sentence, *bare metal* is a compound modifier, and those are usually hyphenated if they appear before a noun. (E.g., "The machines are bare metal." vs "It runs on bare-metal machines.")

Reply:
That makes sense, thanks for elaborating. Could you apply the same change in other relevant places?

Reply:
Thanks!